StyleCop.Analyzers/StyleCop.Analyzers/DocumentationRules/SA1600ElementsMustBeDocumented.cs

```diff
@@ -116,6 +121,10 @@ public void HandleMethodDeclaration(SyntaxNodeAnalysisContext context)
                 }
 
                 MethodDeclarationSyntax declaration = (MethodDeclarationSyntax)context.Node;
+                if (!(declaration.Parent is BaseTypeDeclarationSyntax))
```

Comment:
If that method is not allowed to throw an exception, what would the result be? I think this is a legitimate reason for throwing an exception which is why I solved the problem in the calling code.